PSReadLine/Prediction.Views.cs

```diff
@@ -323,7 +386,11 @@ internal override void GetSuggestion(string userInput)
 
                     if (UseHistory)
                     {
-                        _listItems = GetHistorySuggestions(userInput, ListMaxCount);
+                        _listItems = GetHistorySuggestions(userInput, HistoryMaxCount);
+                        if (_listItems?.Count > 0)
+                        {
+                            _sources = new List<SourceInfo>() { new SourceInfo(SuggestionEntry.HistorySource, _listItems.Count - 1, -1) };
```

Comment:
This is going to set `ItemCount = EndIndex - PrevSourceEndIndex;` in the initializer of `SourceInfo` for `_listItems.Count - 1 - -1`, which I think is what you're intending since the double negative of `PrevSourceEndIndex` cancels out the -1 applied to `Count`, and so we get back the original `_listItems.Count`. Just had to walk myself through it to catch that double-negative (thought at first we were subtracting 2 instead).

Reply:
Maybe add `prevSourceEndIndex:` to name the argument like you do below, just for clarification really.

```suggestion
                            _sources = new List<SourceInfo>() { new SourceInfo(SuggestionEntry.HistorySource, _listItems.Count - 1, prevSourceEndIndex: -1) };
```

Reply:
Will make it a named parameter.